songs/die-aerzte/die-perfekte-diktatur.tex

```diff
@@ -0,0 +1,96 @@
+\beginsong{Die perfekte Diktatur}[by={Die Ärzte}]
```

Comment:
I am not that deep in the Ärzte game but shouldn't this be `by={Farin Urlaub Racing Team}`.
For me it's not a big deal, but for others it might be, so IMO this should be changed.
Reference: https://de.wikipedia.org/wiki/Farin_Urlaub

Note: If we decide to change this, we should also update the parent folder.